README.md

```diff
@@ -4,25 +4,54 @@ The Quick Look Framework (QLF) is part of the DES Instrument Control System (ICS
 
 QLF current version allows to follow the execution of QL pipeline, which process multiple cameras/arms in parallel. The interfaces for QA display are now in a mature stage of development using React and Bokeh plots.
 
-### Clone Quick Look Framework Project
+_Requires ~5 GB_
 
```

Comment:
<Hi Machado, one can infer this step, but just in case, we could add:>
mkdir qlf